src/react-components/src/components/02-molecules/SearchBar/SearchBar.tsx

```diff
@@ -67,7 +86,11 @@ export default function SearchBar(props: SearchBarProps) {
     iconDecorative: true,
   };
 
-  return <form className={bem(searchbar__base_class, modifiers)} id={searchBarId} aria-label="Search for keyword, author, title, or subject">
+  return <form className={`${bem(searchbar__base_class, modifiers, blockName)} ${bem(searchbar__base_class, modifiers)}`}
+    id={searchBarId}
+    role="search"
+    aria-label={searchBarAriaLabel}
+    aria-labelledby={searchBarAriaLabelledBy}>
```

Comment:
Is this a case where you need either but not both?

Reply:
Yes, but it should take labelledBy rather than reject it outright if both happen to be passed.  I'll put in test cases to reflect this.

Reply:
You can create a new object to destruct the props so only one attribute is added to the element.

Reply:
Can you link me to an example of this?  This is one of those nifty things in React that I'd love to do and don't know how to haha ... it'll be cleaner than this: https://github.com/NYPL/nypl-design-system/pull/65/files#diff-22a3559aa6a3f1be78ca9ee249fb4172R29

Reply:
Yeah, that's what I was thinking:
```
let formProps = {};
if (searchBarAriaLabel) {
  formProps['aria-label'] = searchBarAriaLabel;
} else ....

<form {...formProps} />
```

or `"aria-label": !ariaLabelledBy ? ariaLabel : undefined,`.

Reply:
ah darn, I thought there was some nifty react syntax to tell React "one of these things must be passed"